LICENSE.txt

```diff
@@ -627,3 +627,13 @@ Apache 2.0
 https://github.com/JoshClose/CsvHelper/blob/master/LICENSE.txt
 
 Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+MimeMapping v2.0.0
```

Comment:
guess we need to add this to the third parties wiki

Reply:
Yes, I've done so already (if you mean the spreadsheet).